sails/macros/Cargo.toml

```diff
@@ -15,7 +17,7 @@ proc-macro-error.workspace = true
 
 [dev-dependencies]
 parity-scale-codec = { workspace = true, features = ["derive"] }
-sails.workspace = true
+sails = { path = "..", package = "sails-rs" }
```

Comment:
Why?

Reply:
has no effect overall on testing, debugging, developing, but makes possible to publish: there's cargo and crates-io things with dependencies of publishing crates. It makes it really easier